### ISSUE TITLE
[5.0] Incorrect binding of glTexImage3D()

### ISSUE THREAD

**Issue by utkumaden** [opentk/opentk#1392]
### Description
glTexImage3D() is bound incorrectly in OpenTK 5.0-pre6.

The parameter of the function `internalFormat` is generated as `(int)` rather then the expected enumeration `(PixelFormat)`.

### Repro steps
Invoke `GL.TexImage3D()`

### Expected Function Signature
Function signature of the current binding:
```cs
public static unsafe void TexImage3D(
      TextureTarget target,
      int level,
      PixelFormat internalformat,
      int width,
      int height,
      int depth,
      int border,
      PixelFormat format,
      PixelType type,
      void* pixels)
```

### Actual Function Signature
```cs
public static unsafe void TexImage3D(
      TextureTarget target,
      int level,
      int internalformat,
      int width,
      int height,
      int depth,
      int border,
      PixelFormat format,
      PixelType type,
      void* pixels)
```

### Related information 

* Operating system: Any
* What version of OpenTK: Nuget 5.0-pre6
* What runtime: .NET 5
* Workarounds (if any): Cast enumeration of type `PixelFormat` to `int`, e.g. `(int)PixelFormat.Rgba`

**Comment by utkumaden** [comment]
I should add that other overloads such as `IntPtr` and `Span<>` for the parameter `pixels` is also missing.

**Comment by frederikja163** [comment]
[message (16).txt](https://github.com/opentk/opentk/files/8312057/message.16.txt)
[malplaced_groups_correct.txt](https://github.com/opentk/opentk/files/8312058/malplaced_groups_correct.txt)
Other possible solution

When looking at a GLint with a group tag, we check all our enum groups to see if there is a matching enum group name. If we find one we might be able to assume they are the same enum groups?

**Comment by NogginBops** [comment]
Moving this to OpenTK 5-pre8 as we want to get pre7 out and there have been no work done on this. This is definitely something we still want to fix though.

**Comment by NogginBops** [comment]
This is fixed with #1493
